notebooks/dan_kositzke_notebook_testing.ipynb

```diff
@@ -0,0 +1,1651 @@
+              "          dataTable['output_type'] = 'display_data';\n",
+              "          await google.colab.output.renderOutput(dataTable, element);\n",
+              "          const docLink = document.createElement('div');\n",
+              "          docLink.innerHTML = docLinkHtml;\n",
+              "          element.appendChild(docLink);\n",
+              "        }\n",
+              "      </script>\n",
+              "    </div>\n",
+              "  </div>\n",
+              "  "
+            ],
+            "text/plain": [
+              "         id            name    language\n",
+              "0  33247344  Lesa Valcarcel      Python\n",
+              "1  27674388   Tommy Drennon        Java\n",
+              "2  14136744    John Stanley  TypeScript\n",
+              "3  19281044     Jane Gordon  JavaScript\n",
+              "4  45996550    David Fulton  JavaScript"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 456
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# dataframe language distribution matches chart above (more or less)\n",
+        "mentee_df[\"language\"].value_counts(normalize=True)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "VmM4BxSKcDdm",
+        "outputId": "2b3d6c73-1663-4f5d-ed13-6abe387e43b6"
+      },
+      "execution_count": 457,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "JavaScript    0.12\n",
+              "TypeScript    0.11\n",
+              "Java          0.10\n",
+              "Bash/Shell    0.09\n",
+              "C++           0.08\n",
+              "HTML/CSS      0.08\n",
+              "Python        0.07\n",
+              "SQL           0.06\n",
+              "Go            0.04\n",
+              "C#            0.04\n",
+              "PHP           0.04\n",
+              "Node.js       0.04\n",
+              "PowerShell    0.02\n",
+              "Kotlin        0.02\n",
+              "Assembly      0.02\n",
+              "Dart          0.02\n",
+              "Matlab        0.01\n",
+              "Swift         0.01\n",
+              "C             0.01\n",
+              "Ruby          0.01\n",
+              "R             0.01\n",
+              "Name: language, dtype: float64"
+            ]
+          },
+          "metadata": {},
+          "execution_count": 457
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Create RandomMentor Class and Dataframe"
+      ],
+      "metadata": {
+        "id": "7kxIZ6wt5FEz"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def get_unique_languages():\n",
+        "  \"\"\"\n",
+        "  Generates a list of 3 languages for each random mentor.\n",
```

Comment:
When you say random here, are you saying that you're not incorporating the percentages from statista_data? If that is the case, I would encourage consistency between the mentee's and mentor's for more accurate results?